src/main/java/com/powsybl/openloadflow/util/LfHvdcUtils.java

```diff
@@ -0,0 +1,36 @@
+package com.powsybl.openloadflow.util;
+
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.iidm.network.HvdcConverterStation;
+import com.powsybl.openloadflow.network.LfNetwork;
+
+public final class LfHvdcUtils {
+
+    private LfHvdcUtils() {
+    }
+
+    public static boolean isHvdcDandlingInIidm(HvdcConverterStation<?> station, LfNetwork network) {
+
+        if (isIsolated(station.getTerminal().getBusBreakerView().getBus(), network)) {
+            return true;
+        } else {
+            return station.getOtherConverterStation().map(otherConverterStation -> {
+                Bus bus = otherConverterStation.getTerminal().getBusView().getBus();
+                return isIsolated(bus, network);
+            }).orElse(true); // it means there is no HVDC line connected to station
+        }
+    }
+
+    private static boolean isIsolated(Bus bus, LfNetwork network) {
+        if (bus == null) {
+            return true;
+        }
+        if (network != null && network.getBusById(bus.getId()) != null) {
+            // connectivity for that bus to be determined by LfNetwork
+            return false;
+        }
+        // Isolated if only connected to the station
+        return bus.getConnectedTerminalCount() == 1;
```

Comment:
It does not work for iidm network in node/breaker view because a bus bar section has a `Terminal`.

Reply:
@annetill would it work with the same criteria if instead of taking a Bus as Argument we took a Terminal
and defined the bus with
private static boolean isIsolated(Terminal t, LfNetwork network) {
t.getBusBreakerView().getBus()

Reply:
I see that dynaflow uses a isBeaker parameter and can work in either mode. Ill try to get aligned with it and with its criteria to disable a bus.

Reply:
The criteria we have chosen for the #961 is that the bus is in a component with only one bus. It is a simple criteria, that will lead to issue with wind off shore through hvdc line, but I thought it was what we have decided. When I see your criteria, I think that you want to say that only the VSC converter station is connected to the bus (better, even if a bus bar section filter must be done). So what we do?